CONTRIBUTING.md

```diff
@@ -29,19 +25,73 @@ for signing our CLA](https://www.citusdata.com/community/CLA)
    brew link openssl --force
    ```
 
-3. Get the code
+3. Get, build and test the code
 
    ```bash
    git clone https://github.com/citusdata/citus.git
+
+   cd citus
+   ./configure
+   make
+   sudo make install
+   cd src/test/regress
+   make check
    ```
 
-4. Build and test
+#### Linux with Apt-Get (Ubuntu, Debian)
+
+1. Install build dependencies
 
    ```bash
+  sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+  wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+  sudo apt-get update
+
+  apt-get install -y --no-install-recommends \
+          autotools-dev build-essential ca-certificates debhelper devscripts \
+          fakeroot flex libedit-dev libfile-fcntllock-perl libpam0g-dev \
+          libselinux1-dev libxslt-dev lintian postgresql-server-dev-9.5 \
+          postgresql-9.5 git
+   ```
+
+2. Get, build and test the code
+
+   ```bash
+   git clone https://github.com/citusdata/citus.git
    cd citus
    ./configure
    make
    sudo make install
    cd src/test/regress
-   make check-multi
+   make check
+   ```
+
+#### Linux with Yum (RHEL, CentOS, Fedora)
+
+1. Find the Postgres 9.5 RPM URL for your repo at [yum.postgresql.org](http://yum.postgresql.org)
+   (e.g.  for CentOS 7 it is
```

Comment:
This may change (it's already at `9.5-2`, so a `-3` could come alone), so I'm a little worried about having it here, since CentOS users will probably just copy-paste. Is it worth having?